titan/interactions/pca.py

```diff
@@ -28,7 +28,8 @@ def get_num_acts(cls, model: "model.TITAN", rel: "agent.Relationship") -> int:
             if min == max:
                 num_acts = min
             else:
-                num_acts = model.run_random.randrange(min, max)
+                assert not isinstance(min, int)
```

Comment:
shouldn't this be an int? is this a limitation of uniform? if so, we should create a util wrapper to deal with that.